packages/manager/modules/web-paas/src/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "web_paas_title": "Vos projets Web PaaS powered by Platform.sh",
+  "web_paas_title": "Web PaaS powered by Platform.sh",
```

Comment:
**nitpick(non-blocking):** Looks like static content. Not sure that we should rely on translation files. Is there any chance to move it elsewhere?